src/main/java/io/swagger/codegen/languages/java/JavaInflectorServerCodegen.java

```diff
@@ -35,7 +35,12 @@ public JavaInflectorServerCodegen() {
         super();
         sourceFolder = "src/gen/java";
         apiTestTemplateFiles.clear(); // TODO: add test template
-        embeddedTemplateDir = templateDir = String.format("%s/JavaInflector", DEFAULT_TEMPLATE_VERSION);
+        String templateVersion = getTemplateVersion();
+        if (StringUtils.isNotBlank(templateVersion)) {
+            embeddedTemplateDir = templateDir = String.format("%s/JavaInflector", templateVersion);
+        } else {
+            embeddedTemplateDir = templateDir = String.format("%s/JavaInflector", DEFAULT_TEMPLATE_VERSION);
+        }
```

Comment:
@jmini how about if we move this logic to `processOpts` method, just as we did on `JavaClientCodegen`?

Reply:
+1

I missed it during the review. I will have an other look at it.